<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SignOnPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PaymentPushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="MessageTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>451</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitPushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect to Reader</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
